.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,53 @@
+name: Pull Request Build
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
```

Comment:
Can we remove `closed` from this list? Could we then also remove all the `if != closed` statements throughout the action?

Reply:
We can't because otherwise every surge preview sticks around forever. Which maybe isn't a huge deal, but it feels tidier to delete them when a PR is closed (or merged). I think I read that surge accounts have a quota of 500 sites.

Reply:
> it feels tidier to delete them when a PR is closed (or merged).

Agree - totally makes sense!